Automattic/vip-block-data-api
# Remove analytics tracking for non-vip sites

## Description

This will resolve #40 by turning off analytics tracking for non-vip sites. This way if you aren't on VIP there's no opt out to be done.

Follow up discussion is in the works to see if we can allow opt out/in for VIP sites as well. Getting this in first is important to ensure non vip sites don't have analytics being sent.

## Steps to Test

1. Check out PR.
2. Spin up a wp-env and ensure nothing is sent
3. Turn off the VIP sites check and set the keys to have -test at the end
4. Spin up a vip dev-env with this and ensure stats are sent
